documentation/configuration/parameters/target.md

```diff
@@ -9,7 +9,17 @@ redirect_from: /documentation/configuration/target/
 # Target
 
 ## Description
-The target version up to which Flyway should consider migrations. Migrations with a higher version number will be ignored. If set to a value other than `current` or `latest`, this must be a valid migration version (e.g. `2.1`).
+
+The target version up to which Flyway should consider migrations. If set to a value other than `current` or `latest`, 
+this must be a valid migration version (e.g. `2.1`).
+
+When migrating forwards, Flyway will apply all migrations up to and including the target version. Migrations with a 
+higher version number will be ignored. If the target is <code>current</code>, then no versioned migrations will be
```

Comment:
should this not be 
`code`
as per above? I guess html would work here but we should probably stick to markdown when its available.